Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -210,21 +244,15 @@ def test_cleanup
   end
 
   def test_readall
-    repo = CoreTap.new
-    formula_file = repo.formula_dir/"foo.rb"
-    formula_file.write <<-EOS.undent
-      class Foo < Formula
-        url "https://example.com/foo-1.0.tar.gz"
-      end
-    EOS
-    alias_file = repo.alias_dir/"bar"
+    include_formula_file
+    alias_file = CoreTap.new.alias_dir/"balltest"
```

Comment:
`balltest` is an alias for the `testball` formula that is being set up here and in the following two lines, if I'm not misreading the code.